data/templates/post-windows-install.ps1

```diff
@@ -0,0 +1,49 @@
+curl http://<%=server%>:<%=port%>/templates/unattend_server2012.xml -Outfile unattend_server2012.xml

```

Comment:
Rather than pulling down the unattended file again and parsing the comments, why not render this template with the VLAN id values already in it?